pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -1211,9 +1219,12 @@ func cosignSecretArgs(secret corev1.Secret) []string {
 	return cosignArgs
 }
 
-func envs(envs []corev1.EnvVar, envVar corev1.EnvVar) []corev1.EnvVar {
-	if envVar.Name != "" && envVar.Value != "" {
-		envs = append(envs, envVar)
+func envs(envs []corev1.EnvVar, envVar1 corev1.EnvVar, envVar2 corev1.EnvVar) []corev1.EnvVar {
+	if envVar1.Name != "" && envVar1.Value != "" {
+		envs = append(envs, envVar1)
+	}
+	if envVar2.Name != "" && envVar2.Value != "" {
+		envs = append(envs, envVar2)
 	}
 	return envs
 }
```

Comment:
Maybe something like this so we can support more than 2 env vars in the future

```suggestion
func envs(envs []corev1.EnvVar, envVars ...corev1.EnvVar) []corev1.EnvVar {
	for _, envVar := range envVars {
		if envVar.Name != "" && envVar.Value != "" {
			envs = append(envs, envVar)
		}
	}
	return envs
}
```

Reply:
I meant to change it and I forgot.
Thanks!